daprdocs/content/en/js-sdk-docs/js-client/_index.md

```diff
@@ -37,11 +37,20 @@ const server = new DaprServer(serverHost, serverPort, daprHost, daprPort, Commun
 const client = new DaprClient(daprHost, daprPort, CommunicationProtocolEnum.GRPC);
 ```
 
-##### DaprClient Library
+{{% alert title="Note" color="primary" %}}  
+By default the example will run using HTTP. To use gRPC, do the following:
+
+- Replace both occurrences of `process.env.DAPR_HTTP_PORT` with `process.env.DAPR_GRPC_PORT`
+- Replace both occurrences of `CommunicationProtocolEnum.HTTP` with `CommunicationProtocolEnum.GRPC`
```

Comment:
Personally, I find this unclear. CommunicationProtocolEnum is an optional field and defaults to HTTP. We should thus have: to utilize gRPC, add CommunicationProtocolEnum.GRPC to the DaprClient object creation

Reply:
I agree! @greenie-msft and I wanted to bring this over to the doc just for clarification and wanted to see what you thought about explicitly mentioning http.